lib/core/models/Config.ts

```diff
@@ -10,4 +10,6 @@ export default interface Config {
   maxConcurrentDownloads: number;
   observingIntervalInSeconds: number;
   mongoDbConnectionString: string;
+  maxNumberOfOperationsPerBlock: number;
+  maxNumberOfTransactionsPerBlock: number;
```

Comment:
Design: these two protocol config parameters definitely should not come from server config file. Move them to ProtocolParameters.ts